freqtrade/analyze.py

```diff
@@ -144,6 +144,9 @@ def get_signal(pair: str, signal: SignalType) -> bool:
     except ValueError as ex:
         logger.warning('Unable to analyze ticker for pair %s: %s', pair, str(ex))
         return False
+    except Exception as inst:
+        logger.warning('Unexpected error when analyzing ticker for pair %s: %s', pair, str(inst))
```

Comment:
I think we should log this broad exception with `logger.exception()` so that we have a stacktrace

Reply:
done.